tests/console/verify_config_files.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Module allows validation of generic configuration files.
+# Test data should be defined in the following format:
+#  configuration_files:
+#    - path: /etc/hosts
+#      entries:
+#        - 'new.entry.de\t10.226.154.19 h999uz'
+#    - path: /etc/chrony.conf
+#      entries:
+#        - pool ntp.suse.de iburst
+# NOTE: grep -P is used for validation, therefore can be used in entries
```

Comment:
s/can be/cannot be/ is what you meant?

Reply:
I understand your confusion here, I miss `perl regex` before, as that's what `-P` does.